pkg/service/catalog.go

```diff
@@ -206,6 +207,11 @@ func (s *service) GetName() string {
 	return s.Name
 }
 
+// GetPlans returns all of the service's plans
+func (s *service) GetBindable() bool {
+	return s.Bindable
+}
```

Comment:
👍 This will be useful.

Reply:
Yeah, everything had just been bindable so far, but I think that will change in the future and this made sense to me. I caught my issue with the Bind() method on the server only case because of the tests. I assumed if it was non-bindable it wouldn't try and bind/unbind, but it was so I plumbed this in.  

Changed the comment locally and haven't pushed.